### PR TITLE
feat: Added wishlist context

### PR DESCRIPTION
**Pull request by TusharJade**
### Added wishlist context with all add and remove from wishlist functionality

### CODE REVIEW

File: src/components/ProductCard/ProductCard.css [hunk 1]
```diff
@@ -26,4 +27,13 @@
...
     color: var(--white);
+}
+
+.wishlist-hover{
+    cursor: pointer;
+}
+
+.heartfull{
+    color: #ff4343;
+
```
Comment (teja538) [inline src/components/ProductCard/ProductCard.css]: use variables for colors

**Review by sashaboi — APPROVED** [review]
Functionality looks really cool , great work 

On the deploy preview , we are taken to the home page , where its not very clear how to navigate to products page. could you maybe write a blanket link tag to the products rendered on home page to take to the products page as well? might help with UX, 
Onkar

**Review by teja538 — COMMENTED** [review]
code looks good and clean
just observed one thing in UI : while clearing the filters - the radio buttons of sort by and rating being checked

File: src/components/ProductCard/ProductCard.css (hunk 1 above)
Comment (TusharJade) [reply src/components/ProductCard/ProductCard.css]: yes in last will do all of that refactor work and make code clean.

**Comment by TusharJade** [comment]
> code looks good and clean
> just observed one thing in UI : while clearing the filters - the radio buttons of sort by and rating being checked

yah forgot that will do it before merging

**Comment by TusharJade** [comment]
> Functionality looks really cool , great work 
> 
> On the deploy preview , we are taken to the home page , where its not very clear how to navigate to products page. could you maybe write a blanket link tag to the products rendered on home page to take to the products page as well? might help with UX, 
> Onkar

Yes, thats the next thing I am going to implement after this per merge, thanks for pointing it out and looking around all features 🙌

**Pull request merged**